src/clip.sol

```diff
@@ -328,6 +328,24 @@ contract Clipper {
         return active[idx];
     }
 
+    // Externally returns boolean for if an auction needs a redo
+    function needsRedo(uint256 id) external view returns (bool) {
+        // Read auction data
+        Sale memory sale = sales[id];
+
+        require(sale.tab > 0, "Clipper/not-running-auction");
```

Comment:
I'm not sure this should `revert`.
I was imagining the return value as:
```
return sale.tab > 0 && done(sale, price);
```

Reply:
Yes you're right I'll update this.